carpetmodSrc/carpet/commands/CommandLazyChunkBehavior.java

```diff
@@ -25,38 +27,101 @@
 
     public String getUsage(ICommandSender sender)
     {
-        return "Usage: lazychunkbehavior <X> <Z>";
+        return "lazychunkbehavior <add, removeAll, remove, list> <X> <Z>";
     }
 
     public String getName()
     {
         return "lazychunkbehavior";
     }
 
+    private void list(ICommandSender sender, String[] args) throws CommandException{
+        if (args.length != 1)
+        {
+            throw new WrongUsageException(getUsage(sender), new Object[0]);
```

Comment:
Varargs means you don't need this parameter
```suggestion
            throw new WrongUsageException(getUsage(sender));
```